cypress/integration/crd.spec.js

```diff
@@ -211,3 +179,29 @@ describe('crd objects page - valid object', () => {
     cy.url().should('include', '/#/customresourcedefinition?namespace=default');
   });
 });
+
+describe('deletes crd', () => {
+  it('deletes crd from menu dropdown', () => {
+    cy.visit('/#/customresourcedefinition');
```

Comment:
Please use menu entries for navigation as it will cover navigation better. Current test doesn't check if the navigation works.

Reply:
Okay, I'll do that.

Reply:
Done.